docs/general/services.md

```diff
@@ -40,7 +53,7 @@ Website: https://botlist.space
   <a href="https://botsfordiscord.com"><h1>Bots For Discord</h1></a>
 
 Keys: `botsfordiscord`, `botsfordiscord.com`, `bfd`  
-Class: [dbots.BotsForDiscord](/#/docs/main/master/class/BotsForDiscord)  
+Class: [dbots.BotsForDiscord](/#/docs/main/bdb/class/BotsForDiscord)  
```

Comment:
This is probably an oversight in the build, but this needs to be fixed.

Reply:
No, it's right: this way docs show up correctly on the website, even for other branches: once it gets merged to master the docs will be updated to show `master` again. [Docs for this branch](https://dbots.js.org/#/docs/main/bdb/general/services)

Reply:
Oh right, forgot about that.